samples/algorithms/oracle-synthesis/README.md

```diff
@@ -17,14 +17,21 @@ algorithms](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.91.027902)
 and A. Aspuru-Guzik [[Efficient quantum circuits for diagonal unitaries without
 ancillas](http://iopscience.iop.org/article/10.1088/1367-2630/16/3/033040/meta),
 *New J. of Phys.* **16**, 033040, 2014].
- 
-## Prerequisites ##
+
+This sample describes in detail the underlying concepts that were used to implement
+the following operations from the [Q# Standard library](https://github.com/microsoft/QuantumLibraries/tree/master/Standard):
+
+- [Microsoft.Quantum.Canon.ApplyAnd](https://docs.microsoft.com/qsharp/api/qsharp/microsoft.quantum.canon.applyand)
+- Microsoft.Quantum.Synthesis.ApplyXControlledOnTruthTable
+- Microsoft.Quantum.Synthesis.ApplyXControlledOnTruthTableWithCleanTarget
```

Comment:
I think these two items can now safely be made into links.

Reply:
True